screenpear/__init__.py

```diff
@@ -28,9 +28,11 @@ def cli():
 @cli.command()
 @click.option('--src', help='')
 @click.option('--dst', help='')
-def ocr(src, dst):
+@click.option('--img_w', help='', default=-1)
```

Comment:
Let's rename the param to `width` and add an alias `-w`
```suggestion
@click.option('--width', '-w' , help='')
```